adm_program/modules/contacts/import_user.php

```diff
@@ -79,7 +79,7 @@
 
         $userImport->setImportMode((int)$_SESSION['user_import_mode']);
         if ($identifyUserByUuid) {
-            $userImport->readDataByUuid($line[$importProfileFields['usr_uuid']]);
+            $userImport->readDataByUuid($line[$importProfileFields['usr_uuid']] ?? '');
```

Comment:
a simple replacement of null values with an empty string solved all errors I was encountering :)